### ISSUE TITLE
indicator-type setting is missing in the ui

### ISSUE THREAD

**Issue by Tranquility** [codito/gnome-pomodoro#262]
I found the option reading the code and changed it using dconf, but it should be exposed in the ui.

**Comment by RichAyotte** [comment]
Related to #238

**Comment by vinhcoder929** [comment]
i run gsetting but receive no such schema, how to fix it ?

> gsettings set org.gnome.pomodoro.plugins.gnome indicator-type text
No such schema 'org.gnome.pomodoro.plugins.gnome'

**Comment by vinhcoder929** [comment]
install with software center, but cant set text indicator  with gsetting and dconf-editor.
im trying install with git repo, but im receive this.

> make
make: *** No targets specified and no makefile found.  Stop.

**Comment by sojusnik** [comment]
What is the rationale behind removing those three indicator types from the settings? Only by accident I stumbled upon the possibility to change them. Otherwise I never would have found out about the short-text indicator.

**Comment by clockworkpc** [comment]
> I found the option reading the code and changed it using dconf, but it should be exposed in the ui.

Agreed.  This is my favourite pomodoro application on Gnome Shell, and now that I now how to change it, it's trivial, but it's ironic to trawl through Github Issues for a command line fix to a desktop app that's supposed to help me save time.

**Comment by kamilprusko** [comment]
Using latest app and [extension](https://github.com/focustimerhq/gnome-shell-extension-focus-timer), it's not in the Appearance section:

<img width="1074" height="626" alt="Image" src="https://github.com/user-attachments/assets/7c049dcc-c1f9-4130-89b8-85655d6d853c" />
